src/labelle/cli/cli.py

```diff
@@ -297,7 +303,18 @@ def run():
         else None
     )
 
-    dymo_labeler = DymoLabeler(tape_size_mm=args.tape_size_mm)
+    is_print = not (
```

Comment:
The name `is_print` is really weird grammatically.

How about `requires_device`?